### ISSUE TITLE
R4 won't play stream video

### ISSUE THREAD

**Issue by srekkas** [xibosignage/xibo-linux#154]
**Describe the bug**
Layout with video from TVHeadend server won't play. But on same OS link plays with Video app.

**To Reproduce**
Create layout with m3u video file. Link to tvheadend server
http://10.20.xx.xx/stream/channel/8a87a213f2a12af2aaae696d76c50448

**Expected behavior**
Video must play from stream.

**Logs**
[GstMediaPlayer] Error from element uridecodebin0: Your GStreamer installation is missing a plug-in.

**Screenshots**
Black screen

**Environment (please complete the following information):**
- Linux Distro: Ubuntu 18.04
- Linux Desktop Environment: Gnome
- How the player was launched: snap
- Player Version:  1.8-R4

**Comment by Stivius** [comment]
Hello @srekkas. Could you provide an example of a similar video? I've tested with a couple of m3u8 streams and it seems to work well.

**Comment by srekkas** [comment]
> Hello @srekkas. Could you provide an example of a similar video? I've tested with a couple of m3u8 streams and it seems to work well.

if i can send private message, i will send link to stream.

**Comment by Stivius** [comment]
You can send to my e-mail which can be found in my GitHub profile.

**Comment by srekkas** [comment]
> You can send to my e-mail which can be found in my GitHub profile.

i sent link to yours e-mail

**Comment by dasgarner** [comment]
There has been some discussion around this ticket and it seems like you might be uploading the `m3u` playlist file to the CMS as a Video Widget? I think you would have had to add it to the valid extensions on the module to do that?

That isn't a use-case we currently support, so we'd look at tagging this as an improvement request in order to support that in the future.

Please can you confirm whether or not that is the case?

**Comment by srekkas** [comment]
Yes i was uploading m3u to video widget. But i dont remember if i added m3u
extension, where i can check that?

2019-12-17, an, 15:26 Dan Garner <notifications@github.com> rašė:

**Comment by dasgarner** [comment]
You can go to Modules -> Video Widget -> Edit and see the valid extensions on the form.
As far as I know uploading the m3u file directly won't work on any Player - can I reclassify this as a feature request?

**Comment by srekkas** [comment]
wmv,avi,mpg,mpeg,webm,mp4,m3u,m4v

right now i dont really need m3u, when local video widget works. But that
feature wont hurt, especially if player plays all files/links from m3u,
maybe even with defined order, sequential, random, backward or so.
Thanks for great support :)



2019-12-17, an, 18:58 Dan Garner <notifications@github.com> rašė:

**Comment by dasgarner** [comment]
Thanks for the confirmation and help you've given us - I've marked this up as an enhancement request for now.
